src/sync/sync_permission.hpp

```diff
@@ -121,6 +121,10 @@ class Permissions {
 private:
     static SharedRealm management_realm(std::shared_ptr<SyncUser>, const ConfigMaker&);
     static SharedRealm permission_realm(std::shared_ptr<SyncUser>, const ConfigMaker&);
+
+    static ObjectSchema permission_change_schema();
```

Comment:
Is there any reason for these to be static methods? Having them be inline in `management_realm()` seems fine to me, but if you feel having them as separate methods is preferable then they could just as easily be in static functions in `sync_permission.cpp` rather than static methods on the `SyncPermission` type.

Reply:
Hmm...I think inline probably works for us. We never refer back to the `ObjectSchema` objects directly; our API uses lookup by the object name, so breaking them out into separate methods isn't buying us anything.